deploy/template.yaml

```diff
@@ -6,7 +6,72 @@ parameters:
+        "rootfs_url": "https://mirror.openshift.com/pub/openshift-v4/x86_64/dependencies/rhcos/4.6/4.6.8/rhcos-live-rootfs.x86_64.img",
+        "version": "46.82.202012051820-0"
+      },
+      {
+        "openshift_version": "4.7",
+        "cpu_architecture": "x86_64",
+        "url": "https://mirror.openshift.com/pub/openshift-v4/x86_64/dependencies/rhcos/4.7/4.7.33/rhcos-4.7.33-x86_64-live.x86_64.iso",
+        "rootfs_url": "https://mirror.openshift.com/pub/openshift-v4/x86_64/dependencies/rhcos/4.7/4.7.33/rhcos-live-rootfs.x86_64.img",
+        "version": "47.84.202109241831-0"
+      },
+      {
+        "openshift_version": "4.8",
+        "cpu_architecture": "x86_64",
+        "url": "https://mirror.openshift.com/pub/openshift-v4/x86_64/dependencies/rhcos/4.8/4.8.14/rhcos-4.8.14-x86_64-live.x86_64.iso",
+        "rootfs_url": "https://mirror.openshift.com/pub/openshift-v4/x86_64/dependencies/rhcos/4.8/4.8.14/rhcos-live-rootfs.x86_64.img",
+        "version": "48.84.202109241901-0"
+      },
+      {
+        "openshift_version": "4.9",
+        "cpu_architecture": "x86_64",
+        "url": "https://mirror.openshift.com/pub/openshift-v4/x86_64/dependencies/rhcos/4.9/4.9.0/rhcos-4.9.0-x86_64-live.x86_64.iso",
+        "rootfs_url": "https://mirror.openshift.com/pub/openshift-v4/x86_64/dependencies/rhcos/4.9/4.9.0/rhcos-live-rootfs.x86_64.img",
+        "version": "49.84.202110081407-0"
+      },
+      {
+        "openshift_version": "4.9",
+        "cpu_architecture": "arm64",
+        "url": "https://mirror.openshift.com/pub/openshift-v4/aarch64/dependencies/rhcos/4.9/4.9.0/rhcos-4.9.0-aarch64-live.aarch64.iso",
+        "rootfs_url": "https://mirror.openshift.com/pub/openshift-v4/aarch64/dependencies/rhcos/4.9/4.9.0/rhcos-4.9.0-aarch64-live-rootfs.aarch64.img",
+        "version": "49.84.202110080947-0"
+      },
+      {
+        "openshift_version": "4.10",
+        "cpu_architecture": "x86_64",
+        "url": "https://mirror.openshift.com/pub/openshift-v4/x86_64/dependencies/rhcos/4.10/4.10.3/rhcos-4.10.3-x86_64-live.x86_64.iso",
+        "rootfs_url": "https://mirror.openshift.com/pub/openshift-v4/x86_64/dependencies/rhcos/4.10/4.10.3/rhcos-4.10.3-x86_64-live-rootfs.x86_64.img",
+        "version": "410.84.202201251210-0"
+      },
+      {
+        "openshift_version": "4.10",
+        "cpu_architecture": "arm64",
+        "url": "https://mirror.openshift.com/pub/openshift-v4/aarch64/dependencies/rhcos/4.10/4.10.3/rhcos-4.10.3-aarch64-live.aarch64.iso",
+        "rootfs_url": "https://mirror.openshift.com/pub/openshift-v4/aarch64/dependencies/rhcos/4.10/4.10.3/rhcos-4.10.3-aarch64-live-rootfs.aarch64.img",
+        "version": "410.84.202201251210-0"
+      },
+      {
+        "openshift_version": "4.11",
+        "cpu_architecture": "x86_64",
+        "url": "https://mirror.openshift.com/pub/openshift-v4/x86_64/dependencies/rhcos/pre-release/latest-4.11/rhcos-live.x86_64.iso",
```

Comment:
This doesn't point to a specific image.
This means that it might be difficult to force an upgrade of this image.

Would it be better to pin to a particular image link like this one?
https://mirror.openshift.com/pub/openshift-v4/x86_64/dependencies/rhcos/pre-release/4.11.0-0.nightly-2022-04-16-163450/

Reply:
We had this discussion here: https://github.com/openshift/assisted-service/pull/3862#discussion_r884284295
I checked the history from back when 4.10 got in, and it seems like pinned nightly images were removed after a while (see https://github.com/openshift/assisted-service/commit/94ce200b59e8bb3fa0032819a727aa85b532dc22)

If we can pin it to a particular image that is also persistent then it will be better

Reply:
That's what I'm suggesting.
Look at my comment from the problem PR https://github.com/openshift/assisted-service/pull/3196#discussion_r785057710 and @danielerez comment in the new one https://github.com/openshift/assisted-service/pull/3862#discussion_r884682147

The problematic link is `/pre-release/latest-4.11/<some-specific-image>`
We're suggesting `/pre-release/<specific-version>/<specific-image>`

Reply:
Done
Please take another look